target/linux/ath79/dts/ar9342_ubnt_nanobeam-ac-gen2.dts

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
This is deprecated. Please use GPL-2.0-only or GPL-2.0-or-later:
https://spdx.org/licenses/